NuGet.Config

```diff
@@ -5,5 +5,6 @@
         <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
         <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
         <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
+        <add key="xunit" value="https://www.myget.org/F/xunit/api/v3/index.json" />
```

Comment:
Eventually, we'll be able to remove this. At the moment, the xUnit build needed to make VS 2017 work is on MyGet.